src/FamilyHubs.ServiceDirectory.Core/Postcode/Model/PostcodeInfo.cs

```diff
@@ -1,21 +1,35 @@
-﻿using System.Text.Json.Serialization;
+﻿using Newtonsoft.Json;
```

Comment:
Why the need for the old `NewtonSon.Json`, rather than the built in .Net json support?